cmd/deploy.go

```diff
@@ -0,0 +1,81 @@
+// Package cmd for handling command line arguments
+package cmd
+
+import (
+	"context"
+	"os"
+	"os/exec"
+
+	"github.com/rawdaGastan/gridify/internal/config"
+	"github.com/rawdaGastan/gridify/internal/deployer"
+	"github.com/rs/zerolog"
+	"github.com/rs/zerolog/log"
+	"github.com/spf13/cobra"
+)
+
+// deployCmd represents the deploy command
+var deployCmd = &cobra.Command{
+	Use:   "deploy",
+	Short: "Deploy the project in the current directory on threefold grid",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		ports, err := cmd.Flags().GetUintSlice("ports")
+		if err != nil {
+			log.Error().Err(err).Send()
+			return err
+		}
+
+		config, err := config.LoadConfigData()
+		if err != nil {
+			log.Error().Err(err).Msg("failed to load configuration try logging again")
+			return err
+		}
+
+		repoURL, err := exec.Command("git", "config", "--get", "remote.origin.url").Output()
```

Comment:
we assumed that git exists on the system (should be added as a requirement in the readme too)